src/AutoRest.CSharp/Mgmt/Decorator/MethodExtensions.cs

```diff
@@ -33,7 +33,7 @@ public static bool IsListMethod(this RestClientMethod method, [MaybeNullWhen(fal
 
             if (returnType.IsFrameworkType || returnType.Implementation is not SchemaObjectType)
             {
-                if (returnType.FrameworkType == typeof(IReadOnlyList<>))
+                if (returnType.FrameworkType == typeof(IReadOnlyList<>) || returnType.FrameworkType == typeof(IList<>))
```

Comment:
We have a static method `TypeFactory.IsList` to test if a CSharpType is `IList` or `IReadOnlyList`.
Could we switch to that to make here more readable?

Reply:
updated